microshift_networking/microshift-cni.adoc

```diff
@@ -70,3 +64,11 @@ include::modules/microshift-nw-components-svcs.adoc[leveloffset=+1]
 Bridge mappings allow provider network traffic to reach the physical network. Traffic leaves the provider network and arrives at the `br-int` bridge. A patch port between `br-int` and `br-ex` then allows the traffic to traverse to and from the provider network and the edge network. Kubernetes pods are connected to the `br-int` bridge through virtual ethernet pair: one end of the virtual ethernet pair is attached to the pod namespace, and the other end is attached to the `br-int` bridge.
 
 include::modules/microshift-nw-topology.adoc[leveloffset=+1]
+
+[id="_additional-resources_microshift-cni_{context}"]
+== Additional resources
+
+* xref:../microshift_configuring/microshift-using-config-tools.adoc#microshift-using-config-tools_microshift-config-yaml[Using a YAML configuration file]
+* xref:../microshift_networking/microshift-networking-settings.adoc#microshift-config-OVN-K_microshift-networking[Understanding networking settings]
```

Comment:
```suggestion
* xref:../microshift_networking/microshift-networking-settings.adoc#microshift-config-OVN-K_microshift-networking[Understanding networking settings]
```
When I click this link it takes me down the page to Creating an 'OVN-Kubernetes configuration file' can you make it so it leads to the top level of the page that is 'Understanding network settings?'